clients/cli/Cargo.toml

```diff
@@ -18,13 +18,25 @@ serde_derive = "1.0.103"
 serde_json = "1.0.140"
 serde_with = "3.12.0"
 solana-account-decoder = "2.2.0"
+solana-borsh = "2.2.0"
 solana-clap-v3-utils = "2.2.0"
 solana-cli-config = "2.2.0"
 solana-cli-output = "2.2.0"
 solana-client = "2.2.0"
+solana-commitment-config = "2.2.0"
+solana-instruction = "2.2.0"
+solana-keypair = "2.2.0"
 solana-logger = "2.2.0"
+solana-native-token = "2.2.0"
+solana-pubkey = "2.2.0"
 solana-remote-wallet = "2.2.0"
-solana-sdk = "2.2.1"
```

Comment:
I set all the dependencies to `v2.2.0`, but given that `solana-sdk` was `2.2.1`, I'm second guessing on whether I should bump all of them up to `2.2.1`.